source/code-generation/query-converter/test-converted-queries/install-query-runner.txt

```diff
@@ -0,0 +1,94 @@
+.. _rm-install-query-runner:
+
+====================
+Install Query Runner
+====================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+To use Query Converter Test Bench, Relational
+Migrator must have access to the Query Runner. 
+Query Runner is a Docker container with 
+all the dependencies needed to test your converted 
+queries.
+
+About this Task
+---------------
+
+- Query Runner operates on the ``http`` protocol and 
+  is not encrypted. All data is processed within the 
+  container and no connection details or sensitive data 
+  is exposed during the Query Runner connection process.
+
+- Query Runner can run either locally or remotely.
+
+Before you Begin
+----------------
+
+Install `Docker <https://www.docker.com/get-started/>`__.
+
+Steps
+-----
+
+.. procedure::
+   :style: normal
+
+   .. step:: Get the Query Runner Image
+      
+      .. code-block:: sh
+
+         docker pull public.ecr.aws/v4d7k6c9/relational-migrator-query-runner
+
+   .. step:: Run the Image as a Container
+
+         - To run the container locally:
+         
+            .. code-block:: sh
+
+               docker run --name relational-migrator-query-runner -p 6080:6080 public.ecr.aws/v4d7k6c9/relational-migrator-query-runner
+
+         - To run the container remotely:
+         
+            Add the server address where the  
+            service is running to the ``migrator.language-runner.server.address`` in the 
+            ``user.properties``.
+
+            For example:
+
+            .. code-block:: sh
+
+               migrator.language-runner.server.address: http://myserver:6080 
+    
+            .. tip::
+               
+               For details on where the ``user.properties`` file is located, 
+               see :ref:`file-location`. 
+
+   .. step:: Confirm the Query Runner container is running
+
+      In your web browser navigate to the host and port the Query Runner 
+      container is running. For example: 
+      `http://localhost:6080/status <http://localhost:6080/status>`__.
+
+      Check the ``status`` field of the JSON data returned from the 
+      ``status`` endpoint. A status of ``ok`` means the container is
+      running:
+
+      .. code-block:: 
+         :copyable: false
+
+         {
+         "status":"ok",
+         "server":"App Mod Language Runner",
+         "commitHash":"159119004c2bf8c534e15e5895acb9bc1bfb5b8d",
+         "version":"20240627"
+         }
```

Comment:
Can we also add a note that if RM is updated, it may be necessary to update Query Runner to match with a `docker pull`. The RM UI will show a warning banner if query runner is an old version, and an error banner if it's not running at all.

Reply:
Yes I added a note about this situation in the About this task section so the users can be informed of the possibility at the top of the page.